components/SendMoneyToCollectiveBtn.js

```diff
@@ -40,26 +41,36 @@ class SendMoneyToCollectiveBtn extends React.Component {
 
   async onClick() {
     const { currency, amount, fromCollective, toCollective, description, data, LoggedInUser } = this.props;
-    if (!LoggedInUser || !LoggedInUser.canEditCollective(fromCollective) || !get(data, 'Collective')) {
+    if (!LoggedInUser || !LoggedInUser.canEditCollective(fromCollective) || !get(data, 'account')) {
       return;
     }
-    const paymentMethods = get(data, 'Collective.paymentMethods');
+    const paymentMethods = get(data, 'account.paymentMethods');
     if (!paymentMethods || paymentMethods.length === 0) {
       const error = "We couldn't find a payment method to make this transaction";
       this.setState({ error });
       return;
     }
     this.setState({ loading: true });
     const order = {
-      totalAmount: amount,
-      currency,
-      collective: pick(toCollective, ['id']),
-      fromCollective: pick(fromCollective, ['id']),
+      amount: { valueInCents: amount, currency },
+      toAccount: pick(toCollective, ['slug']),
+      fromAccount: pick(fromCollective, ['slug']),
       description,
-      paymentMethod: { uuid: paymentMethods[0].uuid },
+      paymentMethod: { id: paymentMethods[0].id },
+      frequency: 'ONETIME',
     };
     try {
-      await this.props.sendMoneyToCollective({ variables: { order } });
+      await this.props.sendMoneyToCollective({
+        variables: { order },
+        // We need to update the store manually because the response comes from API V2
+        update: store => {
+          const data = cloneDeep(
+            store.readQuery({ query: collectiveBalanceQuery, variables: { slug: fromCollective.slug } }),
+          );
+          data.Collective.stats.balance = 0;
```

Comment:
Can we read from the returned value instead?

In case there is a problem, we'll be able to see it.

Reply:
@znarf : You mean something like `fromCollective.stats.balance` ? We need to get the object from the cache -> clone it and then modify that object. The value that we have (`fromCollective.stats.balance`) is from v1, cloning that object and trying to modify the v2 cache object will not work I think. 🤔

Reply:
If we add `order.fromAccount.stats.balance` to the mutation, we ca do something like:

```es6
update: (store, { data: { createOrder } }) => {
  ...
  data.Collective.stats.balance = createOrder.order.fromAccount.stats.balance;
}
```

Reply:
@Betree : Ah, I see what you mean now, made the change. 👍🏽